config/routes.rb

```diff
@@ -6,6 +6,8 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   get '/shelters', to: 'shelters#index'
   get '/shelters/new', to: 'shelters#new'
+  get '/shelters/:id/reviews/new', to: 'reviews#new'
+  post '/shelters/:shelter_id/reviews', to: 'reviews#create'
```

Comment:
cool workaround with :shelter_id to pass that information automatically to review